ITC205_BaseLine_Code/src/FixBookUI.java

```diff
@@ -1,24 +1,23 @@
 import java.util.Scanner;
 
-
 public class FixBookUI {
 
-	public static enum UI_STATE { INITIALISED, READY, FIXING, COMPLETED };
+	public static enum UIState { INITIALISED, READY, FIXING, COMPLETED };
 
 	private FixBookControl control;
 	private Scanner input;
-	private UI_STATE state;
+	private UIState state;
 
 	
 	public FixBookUI(FixBookControl control) {
 		this.control = control;
 		input = new Scanner(System.in);
-		state = UI_STATE.INITIALISED;
+		state = UIState.INITIALISED;
 		control.setUI(this);
 	}
 
 
-	public void setState(UI_STATE state) {
+	public void setState(UIState state) {
 		this.state = state;
 	}
 

```

Comment:
D3 - Line 49 & 51 - Change ans -> answer

Reply:
D4 - Line 64 - Delete this line of whitespace between statement and closing brace

Reply:
D5 - Line 67 - Delete this line of whitespace between the closing braces

Reply:
D6 - Line 85 & 86 - Remove these 2 lines of whitespace between the closing braces